Source/Core/Core/PowerPC/JitArm64/JitArm64_FloatingPoint.cpp

```diff
@@ -220,15 +230,20 @@ void JitArm64::fselx(UGeckoInstruction inst)
     m_float_emit.FCMPE(EncodeRegToDouble(VA));
   }
 
-  const bool single = fpr.IsSingle(b, true) && fpr.IsSingle(c, true);
-  const RegType type = single ? RegType::LowerPairSingle : RegType::LowerPair;
-  const auto reg_encoder = single ? EncodeRegToSingle : EncodeRegToDouble;
+  const bool bc_single = fpr.IsSingle(b, true) && fpr.IsSingle(c, true);
```

Comment:
I should draw a line somewhere for which ones I should use a lambda for (since using it for a simple `fpr.IsSingle(a)` is a bit silly), and I think this is simpler enough compared to `fpr.IsSingle(a, !packed) && (!use_b || fpr.IsSingle(b, !packed)) && (!use_c || fpr.IsSingle(c, !packed))` that it makes sense to not use a lambda here. I can rename it, though.